src/agent/CMakeLists.txt

```diff
@@ -143,9 +143,7 @@ target_compile_definitions (
             DO_FILE_USER="${DO_FILE_USER}"
             SYSLOG_FILE_GROUP="${SYSLOG_FILE_GROUP}")
 
-if (ADUC_PLATFORM_LAYER STREQUAL "simulator")
-    target_compile_definitions (${target_name} PRIVATE ADUC_PLATFORM_SIMULATOR)
-else ()
+if (ADUC_PLATFORM_LAYER STREQUAL "linux")
```

Comment:
in all of these instances, please add:

```cmake
else ()
    message (FATAL_ERROR "Invalid platform layer specified: ${ADUC_PLATFORM_LAYER}")
endif ()
```

to avoid hard to find build problems

Reply:
Done